src/App.tsx

```diff
@@ -1,61 +1,124 @@
 import './App.scss';
+import { FC, useState } from 'react';
+import { getNewId, findUserById, preparedTodo } from './helpers';
+import { TodoList } from './components/TodoList/TodoList';
+import usersFromServer from './api/users';
+import { FullTodo } from './types';
 
-// import usersFromServer from './api/users';
-// import todosFromServer from './api/todos';
+export const App: FC = () => {
+  const [todos, setTodos] = useState<FullTodo[]>(preparedTodo);
+
+  const [queryTitle, setQueryTitle] = useState('');
+  const [userId, setuserId] = useState(0);
+
+  const [userValueError, setuserValueError] = useState('');
+  const [titleValueError, settitleValueError] = useState('');
+
+  const selectedUser = (event: React.ChangeEvent<HTMLSelectElement>) => {
+    const { value } = event.target;
+
+    setuserValueError('');
+    setuserId(+value);
+  };
+
+  const handleQueryTitle = (event: React.ChangeEvent<HTMLInputElement>) => {
+    const { value } = event.target;
+
+    settitleValueError('');
+    setQueryTitle(value);
+  };
```

Comment:
I can push todo with spaces in title

![image](https://github.com/mate-academy/react_add-todo-form/assets/84296041/f4fa53c4-4b92-4f09-824d-8f122097073f)